source/algorithms.tex

```diff
@@ -4514,10 +4514,10 @@
 The ranges \range{first}{middle} and \range{middle}{last} shall be
 sorted with respect to \tcode{operator<} or \tcode{comp}.
 \tcode{BidirectionalIterator} shall satisfy the
-\tcode{ValueSwappable} requirements\iref{swappable.requirements}. The type
+\textit{Cpp98ValueSwappable} requirements\iref{swappable.requirements}. The type
```

Comment:
Going forward, we really need to find standardized wording to say "satisfies the syntax constraints" vs. "syntax + semantics", and apply such phrasing uniformly.

Reply:
I'm working on that. My current plan is to uniformly use "`T` meets the CppXXX requirements" for "old" concepts (which is the prevalent library usage, this occurrence is aberrant), "`T` satisfies `Concept`" to mean purely syntactic conformance (which is the proper core language meaning of :"satisfy"), and "`T` models `Concept`" to mean that `T` must both satisfy `Concept` and must further meet the semantic requirements or the program is ill-formed NDR.

I'm working on rewording the combined Ranges paper to use that style, and plan to submit future editorial changes to the rest of the library specification to make everything uniform.

Reply:
See also some earlier discussion in #1263. I understand some decisions are obsolete now.